src/router/graphql.ts

```diff
@@ -29,7 +29,7 @@ if (config.graphql) {
     postgraphiles[db] = postgraphiles[db] ?? {
       lastAccessMs: Date.now(),
       graphile: postgraphile(
-        `postgresql://${config.db.user}:${config.db.password}@${config.db.host}/${db}${config.db.forceSSL ? '?sslmode=require' : ''}`,
+        `postgresql://${config.db.user}:${config.db.password}@${config.db.host}/${db}${config.db.forceSSL ? '?sslmode=no-verify' : ''}`,
```

Comment:
would this mode still force the ssl connection?

Reply:
It forces an ssl connection even if the ssl cert is wrong (which it is, because RDS doesn't let us change it and is why our long-term plan is to run our own Postgres servers instead of using RDS). I thought the prior one would work, but apparently it meant "crash if the cert is bad."